index.html

```diff
@@ -3,9 +3,49 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Bio Page</title>
+    <title>Ylldes Mustafa</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
+    <section id="header" class="card">
+        <h1>Ylldes Mustafa</h1>
+    </section>
+
+    <ul class="side-menu">
+       <li><a href="index.html"> Home</a></li>
```

Comment:
I can't recall if I comment on this in another HTML file, but notice the 'Home' link reloads the 'Home' page without any benefit to the user.  Consider an approach to the 'Home' link which avoids this reload, without changing the appearance of the 'Home' link.